pkg/products/threescale/reconciler.go

```diff
@@ -3078,3 +3088,25 @@ func (r *Reconciler) addSSOReadyAnnotationToUser(ctx context.Context, client k8s
 
 	return nil
 }
+
+func (r *Reconciler) reconcileRatelimitPortAnnotation(ctx context.Context, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	apim := &k8sappsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "threescale-operator-controller-manager-v2",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
```

Comment:
Should this be adding the annotation to the `ApiManager` CR instead ? Changes from https://github.com/3scale/3scale-operator/pull/739/files#diff-58577e8a0e7d8aa700f7595e8b67800c4eab26d20123298924dec34e5f26151bR112-R124 looks to suggest the annotation is added to the `ApiManager` CR rather than the operator deployment :thinking:

Reply:
This was the approach I had taken initially and couldn't get it working.. it would make more sense it being on the cr though, I'll commit my current changes and test this, thanks :)